README.md

```diff
@@ -105,9 +105,34 @@ To make a bundle that is independent of the SystemJS loader entirely, we can mak
 builder.buildStatic('myModule.js', 'outfile.js', options);
 ```
 
-This bundle file can then be included with a `<script>` tag, and no other dependencies would need to be included in the page.
+This bundle file can then be included with a `<script>` tag, and no other dependencies would need to be included in the page. You'll likely want your module
+to export a global variable when loaded from a script tag, and this can be configured via `globalName`.  For example
 
-By default, Traceur or Babel runtime are automatically included in the SFX bundle if needed. To exclude the Babel or Traceur runtime set the `runtime` build option to false:
+```javascript
+builder.buildStatic('src/NavBar.js', 'dist/NavBarStaticBuild.js', { globalName: 'NavBar' });
+```
+
+will cause the output of your module to be assigned to a global variable named `NavBar`.  If you're making a static bundle, while excluding certain dependencies, those dependencies 
+will of course need to have already been loaded on your page, with their own global variables exported.  You can match these global variables up with your needed dependencies 
+with `globalDeps`.  For example
+
+```javascript
+builder.buildStatic('src/NavBar.js - react', 'dist/NavBarStaticBuild.js', { globalName: 'NavBar', globalDeps: { 'react': 'React' } });
```

Comment:
It may be worth documenting `externals` here:

```javascript
builder.buildStatic('src/NavBar.js', 'dist/NavBarStaticBuild.js', {
  externals: ['react'],
  globalName: 'NavBar',
  globalDeps: {
    'react': 'React'
  }
});
```

I also just added https://github.com/systemjs/builder/commit/c3c3e73475ca28e1a7b235ac02338c5d716e0983 so that globalDeps can be treated as externals automatically:

```javascript
builder.buildStatic('src/NavBar.js', 'dist/NavBarStaticBuild.js', {
  globalName: 'NavBar',
  globalDeps: {
    'react': 'React'
  }
});
```